app/src/main/java/com/duckduckgo/app/browser/omnibar/animations/LottiePrivacyShieldAnimationHelper.kt

```diff
@@ -46,7 +46,7 @@ class LottiePrivacyShieldAnimationHelper @Inject constructor(
         val protectedShieldDark: Int
         val unprotectedShield: Int
         val unprotectedShieldDark: Int
-        if (appPersonalityFeature.self().isEnabled() && !appPersonalityFeature.variant1().isEnabled()) {
+        if (!senseOfProtectionExperiment.isUserEnrolledInAVariantAndExperimentEnabled()) {
```

Comment:
This should be (remove the negation):

if (senseOfProtectionExperiment.isUserEnrolledInAVariantAndExperimentEnabled())